include/powsybl/iidm/NetworkIndex.hpp

```diff
@@ -80,6 +81,12 @@ class NetworkIndex {
     mutable IdentifiablesByType m_objectsByType;
 };
 
+template <>
+const Identifiable& NetworkIndex::get(const std::string& id) const;
+
+template <>
+Identifiable& NetworkIndex::get(const std::string& id);
+
```

Comment:
Yes, specialized template should be known from headers with msvc to avoid multiple definition of instanciated templated. If I comment this line :
```
[ 24%] Linking CXX shared library iidm.dll
LINK Pass 1: command "C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1424~1.283\bin\Hostx86\x64\link.exe /nologo @CMakeFiles\iidm.dir\objects1.rsp /out:iidm.dll /implib:iidm.lib /pdb:E:\Projet_SRO\rte\windows-porting-final\cmake-build-debug\src\iidm.pdb /dll /version:0.9 /machine:x64 /debug /INCREMENTAL C:\libxml2-2.9.10\lib\libxml2.lib C:\boost-1.72.0\lib\boost_filesystem.lib C:\boost-1.72.0\lib\boost_system.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\iidm.dir/intermediate.manifest CMakeFiles\iidm.dir/manifest.res" failed (exit code 1169) with the following output:
NetworkIndex.cpp.obj : error LNK2005: "public: class powsybl::iidm::Identifiable const & __cdecl powsybl::iidm::NetworkIndex::get<class powsybl::iidm::Identifiable>(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)const " (??$get@VIdentifiable@iidm@powsybl@@@NetworkIndex@iidm@powsybl@@QEBAAEBVIdentifiable@12@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) déjà défini(e) dans Network.cpp.obj

```